dfanalyzer/graph.py

```diff
@@ -0,0 +1,195 @@
+
+import dask.dataframe as dd
+import re
+import os
+import glob
+import pandas as pd
+
+class GrepIO:
```

Comment:
Can we have a sample dataset (a small set of your existing trace data) in the test/data folder and test this class? This will allow us to test this in the CI as well.